docs/source/install-on-debian.rst

```diff
@@ -0,0 +1,370 @@
+
+        $ sudo apt install nginx
+        .. output ..
+        $
+
+#. Start nginx:
+
+    .. code-block:: console
+
+        $ sudo systemctl start nginx
+        $
+
+#. Visit MYDOMAIN in a browser
+
+    The default nginx page should now be up and running on MYDOMAIN, port 80.
+    Double-check that the page is reachable *worldwide*, that there is no
+    firewall or similar blocking port 80. This is needed for autogenerating the
+    certificate later.
+
+#. Install venv:
+
+    .. code-block:: console
+
+        $ sudo apt install python3-venv
+        .. output ..
+        $
+
+#. Create a user for howitz:
+
+    .. code-block:: console
+
+        $ sudo useradd -m -s /bin/bash howitz
+        $
+
+   This creates a user "howitz" with the group "howitz" which owns the new
+   directory "/home/howitz". The user has (and needs) no password and the shell
+   is set to bash. The "howitz"-user should NOT have root/sudo-access.
+
+#. Become howitz and create a venv:
+
+    .. code-block:: console
+
+        $ sudo su - howitz
+        howitz@myserver:~$ python3 -m venv .venv
+        howitz@myserver:~$
+
+   This creates a directory ".venv"
+
+#. Activate the venv:
+
+    .. code-block:: console
+
+        howitz@myserver:~$ source .venv/bin/activate
+        (.venv) howitz@myserver:~$
+
+   Now only the python libraries installed in the venv are easily available.
+
+
+Get the source code and dependencies
+====================================
+
+Pick one of the below methods:
+
+Use the raw source
+------------------
+
+This is the only way to install the locked dependencies.
+
+#. Install git:
+
+    .. code-block:: console
+
+        $ sudo apt install git
+        .. lots of stuff ..
+        $
+
+#. Switch to howitz-user and activate the venv:
+
+
+    .. code-block:: console
+
+        $ sudo su - howitz
+        howitz@myserver:~$
+        howitz@myserver:~$ source .venv/bin/activate
+        (.venv) howitz@myserver:~$
+
+#. Get source code via git:
+
+    .. code-block:: console
+
+        (.venv) howitz@myserver:~$ git clone https://github.com/Uninett/Howitz.git
+        .. lots of stuff ..
+        (.venv) howitz@myserver:~$
+
+   This creates a directory "Howitz" with the source code.
+
+#. Enter the Howitz directory:
+
+    .. code-block:: console
+
+        (.venv) howitz@myserver:~$ cd Howitz
+        (.venv) howitz@myserver:~/Howitz$
+
+#. Install the locked dependencies:
+
+    .. code-block:: console
+
+        (.venv) howitz@myserver:~/Howitz$ pip install -r requirements-frozen.txt
+        .. lots of stuff ..
+        (.venv) howitz@myserver:~/Howitz$
+
+#. Install howitz into the venv:
+
+    .. code-block:: console
+
+        (.venv) howitz@myserver:~/Howitz$ pip install .
+        .. lots of stuff ..
+        (.venv) howitz@myserver:~/Howitz$
+
+Install via pip
+---------------
+
+TBD
+
+Configure howitz
+================
+
+See howitz's README.rst for how to configure howitz and create a user.
```

Comment:
Not for this PR, I guess, but the README file could stand to have a more in-depth explanation of the double user database paradigm:  I.e. you don't use your Zino user directly in Howitz, but all Howitz users must be mapped to a Zino user....

Reply:
I've put that into this doc and can move it elsewhere later.